include/FileBrowser.h

```diff
@@ -125,6 +125,7 @@ private slots:
 	void activateListItem( QTreeWidgetItem * item, int column );
 	void openInNewInstrumentTrackBBE( void );
 	void openInNewInstrumentTrackSE( void );
+	void openContainingFolder();
```

Comment:
I think putting this at the bottom of this block would be better, so after the `updateDirectory` line.

Reply:
Done with commit b85aef2f33794eb6bd7fa1153f7c411ef14c5b7c.